### ISSUE TITLE
Inconsistant use of colors

### ISSUE THREAD

**Issue by remyleone** [hashicorp/waypoint#706]
**Describe the bug**

The usage of colors seems to be inconsistant here:

<img width="917" alt="Captura de ecrã 2020-10-26, às 17 05 29" src="https://user-images.githubusercontent.com/300834/97197146-94e8d500-17ad-11eb-8977-9c835341bf60.png">

**Steps to Reproduce**

```
# The name of your project. A project typically maps 1:1 to a VCS repository.
# This name must be unique for your Waypoint server. If you're running in
# local mode, this must be unique to your machine.
project = "my-project"

# Labels can be specified for organizational purposes.
# labels = { "foo" = "bar" }

# An application to deploy.
app "web" {
    # Build specifies how an application should be deployed. In this case,
    # we'll build using a Dockerfile and keeping it in a local registry.
    build {
        use "docker" {}

        # Uncomment below to use a remote docker registry to push your built images.
        #
        # registry {
        #   use "docker" {
        #     image = "registry.example.com/image"
        #     tag   = "latest"
        #   }
        # }

    }

    # Deploy to Docker
    deploy {
        use "docker" {}
    }
}
```

**Expected behavior**

Same usage of colors

**Comment by mitchellh** [comment]
This is not a bug. We purposely only highlight the first line of an error as red. 

During the beta period (~50 people) we highlighted the whole error red, but this led to "wall of red" issues. We got a lot of feedback that dozens of lines being red took away from the main error message. We switched to first line which has issues in short error messages as you've seen, but the beta group agreed it was a benefit for long error messages.

We may be able to tweak this slightly so its the first _sentence_ that is red instead, but until there is a concrete path forward I'll close this as "working as intended"
